backend/src/api/users/entities/user.entity.ts

```diff
@@ -13,6 +15,9 @@ class User {
 
   @Column()
   public username: string;
+
+  @OneToOne(() => Stats, (stats) => stats.user)
```

Comment:
:octopus: ok, this is how each user is linked to his game stats, etc

Reply:
Yes, that is TypeORM way of linking entities in tables (https://typeorm.io/#/one-to-one-relations)